lib/dnsprover.js

```diff
@@ -2,7 +2,6 @@ require('babel-polyfill');
 const Oracle = require('./oracle/oracle');
 const Result = require('./dns/result');
 const Verifier = require('./dns/verifier');
-const OracleProver = require('./oracle/oracle_prover');
 
 class DnsProover{
   constructor(provider) {
```

Comment:
can we be consistent here, either there is a space between the ```)``` and ```{```, or there isn't.